src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/utils/CommonUtils.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.personalisation.utils;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import org.apache.commons.lang3.StringUtils;
+
+public final class CommonUtils {
+    private CommonUtils() {
+        // private constructor to prevent sonar warning
+    }
+
+    public static String convertAsylumCaseFeeValue(String amountFromAsylumCase) {

```

Comment:
Is the first time we do this conversion in notifications code?

Reply:
Nope, but usually it is used locally, I decide to make it static(since Im using it in two classes to avoid code duplications check failures)